packages/frontend/core/src/modules/editor/entities/editor.ts

```diff
@@ -157,6 +157,7 @@ export class Editor extends Entity {
         if (params.databaseId && params.databaseRowId) {
           const defaultOpenProperty: DefaultOpenProperty = {
             type: 'database',
+            docId: params.databaseId,
```

Comment:
wrong？

Reply:
unsubscribing cursor

Reply:
anyway，cursor is not me

Reply:
turns out this section is incorrect. the params for dbid/rowid is for scrolling to the current doc's db row, not to opening the doc's db properties